index.js

```diff
@@ -1,29 +1,52 @@
-var path = require('path');
+/* jshint node: true */
+'use strict';
 
 module.exports = {
   name: 'list-view',
-  included: function (app) {
-    this._super.included(app);
-
-    app.import('vendor/list-view.amd.js', {
-      exports: {
-        'list-view/main': ['default'],
-        'list-view/helper': ['default'],
-        'list-view/list_item_view': ['default'],
-        'list-view/list_item_view_mixin': ['default'],
-        'list-view/list_view': ['default'],
-        'list-view/list_view_helper': ['default'],
-        'list-view/list_view_mixin': ['default']
+  treeForVendor: function() {
+
+    var klassy = new this.Funnel('bower_components', {
```

Comment:
No, they should actually not be included in vendor. Those are only necessary while doing addon development. [index.js](index.js) needs to make sure that they're only included when `this.isDevelopingAddon() !== true`

Try the following code in your addon's index.js, if it works, make a PR please :)
```
/* jshint node: true */
'use strict';

module.exports = {
  name: 'list-view',
  treeForVendor: function() {
    if (!this.isDevelopingAddon()) {
       return;
    }
    var klassy = new this.Funnel('bower_components', {
      srcDir: '/klassy/lib',
      files: ['klassy.js'],
      destDir: '/'
    });

    var emberTestHelpers = new this.Funnel('bower_components', {
      srcDir: '/ember-test-helpers/lib',
      include: [/.js$/],
      destDir: '/'
    });

    var es6EmberTestHelpers = new this.transpileModules(this.mergeTrees([klassy, emberTestHelpers]), {
      esperantoOptions: {
        _evilES3SafeReExports: true,
        strict: true
      }
    });

    var emberTestHelpersAMD = this.concatFiles(es6EmberTestHelpers, {
      inputFiles: ['**/*.js'],
      outputFile: '/ember-test-helpers.amd.js'
    });

    var qunitModule = new this.Funnel('bower_components', {
      srcDir: '/ember-qunit-source/lib/ember-qunit',
      include: [/qunit-module.js$/],
      destDir: '/'
    });

    var es6QunitModule = new this.transpileModules(this.mergeTrees([klassy, qunitModule]), {
      esperantoOptions: {
        _evilES3SafeReExports: true,
        strict: true
      }
    });

    var qunitModuleAMD = this.concatFiles(es6QunitModule, {
      inputFiles: ['**/*.js'],
      outputFile: '/ember-qunit-module.amd.js'
    });

    return this.mergeTrees(['vendor', emberTestHelpersAMD, qunitModuleAMD]);
  }
};
```